.github/workflows/ci.yml

```
@@ -0,0 +1,45 @@
+name: node_js CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Setup Nodejs
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12
+
+#    - name: Run Whitelist Audit
+#      run: npm run audit:whitelisted
```

We're trying to get `audit:whitelisted` working together with Renovate. Can we leave this in for now?

```suggestion
    - name: Run Whitelist Audit
      run: npm run audit:whitelisted
```

---

Sure @pshiu , I've added the check back as you suggested. The check will be failing for now so would we merge the PR with the failing check?

---

Thanks Usama! Yep, I will do that, I'll take the responsibility of fixing anything downstream as I don't want you to have to wait for my PR fixing the audits. Thanks!